src/Filters/molecules/FilterInputs.tsx

```diff
@@ -161,21 +172,28 @@ const FilterInputs: React.FC<IFilterInputs> = ({
 
   }, [visibleSearchInputs]);
 
+  const searchInputsElement = renderSearchInputs(searchFilters, visibleSearchInputs, handleVisibleSearch);
+  const datePickersElement = renderDatePickers(datePickerFilters);
+  const dropDownsElement = renderDropdowns(dropdownFilters, showMoreDropdowns, hasShowMore);
+
+  const filterPosition: IFilterPosition = { Searchers: searchInputsElement, DatePickers: datePickersElement, Dropdowns: dropDownsElement };
+
   return (
     <Container {...{ props }}>
-      {renderSearchInputs(searchFilters, visibleSearchInputs, handleVisibleSearch)}
-      {renderDatePickers(datePickerFilters)}
-      {renderDropdowns(dropdownFilters, showMoreDropdowns, hasShowMore)}
+
+      {Array.from(order)?.map(element => filterPosition[element])}
 
       {/* {When the Dev does not initialize hasShowMore as true but has hidden inputs, it will show the add Searcher of the canHide} */}
       {(!hasShowMore || !showMoreDropdowns) && renderAddSearchButtons(searchFilters, visibleSearchInputs, handleVisibleSearch)}
       {hasShowMore && (
-        <FilterButton
-          icon='FilterEllipsis'
-          onClick={toggleShowMore}
-        >
-          {showMoreDropdowns ? showMoreText : showLessText}
-        </FilterButton>)}
+        <FilterButtonBox>
```

Comment:
Why this FilterButtonBox wrapper is added to FilterButton? I don't think this is needed? Any specific reason?

Reply:
Done